apis/test_config.py

```diff
@@ -262,8 +313,7 @@ def from_pytorch(test_name: str, reserved_tpu: bool = True):
     return JSonnetTpuVmTest._from_json_helper(
         test,
         setup=test['tpuSettings']['tpuVmPytorchSetup']
-        # HACK: Extra setup assumes a new shell in home directory
-        + '\ncd ~\n' + test['tpuSettings']['tpuVmExtraSetup'],
+        + test['tpuSettings']['tpuVmExtraSetup'],
```

Comment:
I assume you have tested it on Airflow?

Reply:
I don't remember why I did that, let me revert this back and redo a test.